### PR TITLE
fix: add padding to discipline tags

### PR DESCRIPTION
**Pull request by Ali7040**
issue #879

### CODE REVIEW

File: .yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (vnugent) [inline .yarnrc.yml:1]: Can you remove this file: .yarnrc.yml?  It seems to cause unexpected changes to the yarn.lock file

**Review by vnugent — CHANGES_REQUESTED** [review]
I just left a comment.  Can you fix linting errors?

File: .yarnrc.yml (hunk 1 above)
Comment (Ali7040) [reply .yarnrc.yml:1]: okay.

**Comment by vnugent** [comment]
Can you also fix linting errors?
https://github.com/OpenBeta/open-tacos/actions/runs/5330206526/jobs/9656652729?pr=880

**Comment by vnugent** [comment]
can you also undo changes to `yarn.lock`?

**Pull request closed without merging**